1748212351944	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1748212352042	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1748212398679	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1748212404765	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1748212404765	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1748212404767	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1748212404769	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1748212416705	Sync.SyncScheduler	DEBUG	Woke from sleep.
1748212416706	Sync.SyncScheduler	DEBUG	Online, will sync in 2s.
1748212416706	Sync.SyncScheduler	DEBUG	Next sync in 2000 ms. (why=wake_notification)
1748212416706	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1748212418707	Sync.Service	DEBUG	User-Agent: Firefox/138.0.4 (Linux x86_64) FxSync/1.140.0.20250518015817.desktop
1748212418707	Sync.Service	INFO	Starting sync at 2025-05-26 04:03:38 in browser session QeVl2r7C5Suw
1748212418708	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1748212418712	Sync.Status	INFO	Resetting Status.
1748212418712	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1748212418713	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1748212418716	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/195504406/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1748212418716	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1748212418716	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1748212418716	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1748212418716	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1748212418718	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
